x-pack/test_serverless/api_integration/test_suites/common/scripts_tests/languages.js

```diff
@@ -11,28 +11,38 @@ import { ELASTIC_HTTP_VERSION_HEADER } from '@kbn/core-http-common';
 import { SCRIPT_LANGUAGES_ROUTE_LATEST_VERSION } from '@kbn/data-plugin/common/constants';
 
 export default function ({ getService }) {
-  const supertest = getService('supertest');
   const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  let roleAuthc;
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
 
   describe('Script Languages API', function getLanguages() {
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
+    });
+    after(async () => {
+      await svlUserManager.invalidateApiKeyForRole(roleAuthc);
+    });
     it('should return 200 with an array of languages', () =>
```

Comment:
@pheyos hey boss man, do we really need admin privileges to do this?  seems like overkill.